static/js/containers/FinancialAidCalculator.js

```diff
@@ -56,7 +55,7 @@ const currencySelect = (update, current) => (
 
 const salaryUpdate = R.curry((update, current, e) => {
   let newEdit = R.clone(current);
-  newEdit.income = sanitizeNumberString(20, e.target.value);
+  newEdit.income = e.target.value
```

Comment:
- It dont allow me to enter text and neither show the validation (please enter whole number)
- it dont allow me to add `.xy`
- it when you copy `555.50` into income it change it to `55550`

that the is reason @pdpinch opened this issue